### PR TITLE
feat(wizard): add support for success step

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly/issues/6843

Links:
* Success - https://patternfly-pr-6844.surge.sh/components/wizard/html/success-on-step/
* Success w/ mobile nav expanded - https://patternfly-pr-6844.surge.sh/components/wizard/html/nav-expanded-with-success-mobile/

And FWIW here are the pre-existing error examples
* Error - https://patternfly-pr-6844.surge.sh/components/wizard/html/error-on-step/
* Error w/ mobile nav expanded - https://patternfly-pr-6844.surge.sh/components/wizard/html/nav-expanded-with-error-mobile/

### CODE REVIEW

File: src/patternfly/components/Wizard/wizard--status.hbs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            {{/wizard-nav-item}}
+          {{/wizard-nav-list}}
+        {{/wizard-nav}}
+      {{#> wizard-main}}
+        {{> __wizard-form}}
+      {{/wizard-main}}
+    {{/wizard-inner-wrap}}
+    {{> wizard-footer}}
+  {{/wizard-outer-wrap}}
+{{/wizard}}
```
Comment (mattnolting) [inline src/patternfly/components/Wizard/wizard--status.hbs:70]:
```suggestion
{{/wizard}}

```

File: src/patternfly/components/Wizard/wizard-nav-link-status-icon.hbs [hunk 2]
```diff
@@ -2,5 +2,5 @@
   {{#if wizard-nav-link-status-icon--attribute}}
     {{{wizard-nav-link-status-icon--attribute}}}
   {{/if}}>
-  {{> @partial-block}}
+  <i class="fas fa-{{ternary wizard-nav-item--IsSuccess 'check-circle' 'exclamation-circle'}}" aria-hidden="true"></i>
```
Comment (mattnolting) [inline src/patternfly/components/Wizard/wizard-nav-link-status-icon.hbs:5]: ❤️

File: src/patternfly/components/Wizard/wizard-nav-link.hbs [hunk 3]
```diff
@@ -37,11 +40,9 @@
   {{#if wizard-nav-link--attribute}}
     {{{wizard-nav-link--attribute}}}
   {{/if}}>
-  {{#if wizard-nav-item--IsDanger}}
-    {{#> wizard-nav-link-status-icon}}
-      <i class="fas fa-exclamation-circle" aria-hidden="true"></i>
-    {{/wizard-nav-link-status-icon}}
-  {{/if}}
+  {{#ifAny wizard-nav-item--IsSuccess wizard-nav-item--IsDanger}}
+    {{> wizard-nav-link-status-icon}}
```
Comment (mattnolting) [inline src/patternfly/components/Wizard/wizard-nav-link.hbs:44]: ❤️

**Review by mattnolting — APPROVED** [review]
LPTM 👍

**Review by nicolethoen — APPROVED** [review]
🐎

**Review by srambach — COMMENTED** [review]
Looks like wizard-info was renamed? Otherwise 🧙

**Review by srambach — APPROVED** [review]

**Pull request merged**